.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,64 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [Bug, Feature, Improvement, Other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: This issue concerns the...
```

Minor, for consistency use title case

```suggestion
      label: This Issue Concerns the...
```

---

I don't think titlecase really fits here as this is a sentence